### ISSUE TITLE
MacOS Unusable

### ISSUE THREAD

**Issue by Nygosaki** [replugged-org/replugged#600]
### Describe the bug

Following [this discord support thread on the Replugged server,](https://discord.com/channels/1000926524452647132/1208346302719721482) replugged causes crashes when opening settings, a forum post, or randomly.

### Reproduction steps

1. Install discord on MacOS
2. Plug replugged in

### Expected behavior

Being able to use discord with replugged enabled; not being unable to launch discord without a crash

### Actual behavior

Constant crashes

### Additional information

- I have unplugged and replugged a couple of times with no luck. 
- The crash happen with or without themes or plugins.   
- Video showing the issue: https://imgur.com/a/FMTJiWy.     
- Version info: 
![image](https://github.com/replugged-org/replugged/assets/63132234/e3ba9853-8c61-4c0d-b2c1-6591bc4c4908).    
- One of the logs when crashing through the Options menu: https://cdn.discordapp.com/attachments/1208346302719721482/1208693666907164732/discord.com-1708245334632.log?ex=65e436b2&is=65d1c1b2&hm=cac9aace57d86d5a6f7b07f4fcf2332da638935f27069ba4887a48ddcf03ab65&.     
- Error that popped up immidietly after going to settings and crashing: 
![image](https://github.com/replugged-org/replugged/assets/63132234/7ea29e05-6bd9-4950-b359-c2b6062a5480)
- Error when crashing from DMs: 
![image](https://github.com/replugged-org/replugged/assets/63132234/b2cc22b0-8207-4ece-8717-0fc1a3dfed85)
- Contents of the file mentioned in the first crash under `j.getPredicateSections` ![image](https://github.com/replugged-org/replugged/assets/63132234/4537bdc8-8580-48b4-866b-4bc40d89106f)
With this line being highlighted in green
```js
getPredicateSections() {
            return replugged.coremods.coremods.settings.insertSections(this.props.sections.filter(e=>null == e.predicate || e.predicate()))
        }
 ```
- Entering `replugged.coremods.coremods.settings` and `replugged.coremods.coremods.messagePopover` into the console both return undefined
- If discord will crash, I can see the nitro theme being loaded before the loading screen
- If it will work, I cant see any themes being applied before loading, and custom replugged themes applied while loading
- If I leave the discord that supposed to crash on for long enough without crashing it, it will move to and then get stuck in the loading "connection problems?" screen
- This is a thing that sometimes happens before the crash:
![image](https://github.com/replugged-org/replugged/assets/63132234/ebf97cca-3e55-46dc-9a9e-e59590467d03)
This is the code in question, with the `, i = Error();` being the one that causes the fail:
![image](https://github.com/replugged-org/replugged/assets/63132234/cbf6cba3-2c73-4299-b9b1-62547f0a55b0)

### Please confirm the following

- [X] I checked if Discord and Replugged are up to date
- [X] I made sure this is not related to an external plugin or a theme
- [X] [I searched the issues](https://github.com/replugged-org/replugged/issues) to make sure this bug has not been reported before.

**Comment by Nanorithm** [comment]
I'm getting the same issue. 100% unusable at the moment.

<img width="556" alt="image" src="https://github.com/replugged-org/replugged/assets/13856563/fc7769f0-4dc2-48e8-adba-0276dc544a74">

**Comment by Nygosaki** [comment]
@Nanorithm just wanted to let you know my current solution: keep restarting discord until it doesn't crash, and then never quit discord or restart your mac

**Comment by FedeIlLeone** [comment]
This issue has been fixed as of v4.8.1. We have improved the stability of the mod, including those errors.
If you still have issues with crashes, try removing some plugins that could cause them.
